core/coreapi/interface/dag.go

```diff
@@ -4,21 +4,38 @@ import (
 	"context"
 	"io"
 
-	options "github.com/ipfs/go-ipfs/core/coreapi/interface/options"
+	"github.com/ipfs/go-ipfs/core/coreapi/interface/options"
 
 	ipld "gx/ipfs/QmZtNq8dArGfnpCZfx2pUNY7UcjGhVp5qqwQ4hH6mpTMRQ/go-ipld-format"
 )
 
-// DagAPI specifies the interface to IPLD
-type DagAPI interface {
+// DagOps groups operations that can be batched together
+type DagOps interface {
```

Comment:
We will probably also have `Delete` on this interface, right?

Reply:
Well, there isn't `dag rm` so I didn't add it here. But yes, that would be the place for that (will be in Block API, but that's for a separate PR)